driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java

```diff
@@ -42,8 +42,11 @@ public ReadTimeoutException(
     super(
         endPoint,
         String.format(
+            "Cassandra timeout during read query at consistency %s (%s). "
+                + "In case this was generated during read repair, the consistency level is not representative of the actual consistency.",
             "Cassandra timeout during read query at consistency %s (%s)",
```

Comment:
this is a redundant line. I believe this line is supposed to be replaced by the new changes.

Reply:
Thanks for pointing out. Probably skipped while switching/raising multiple PR on this. Taken care now.